chapters/modules/chapter_8/pages/sharp.adoc

```diff
@@ -0,0 +1,78 @@
+
+Exponential amortization means that as the computational load of the proofs increases, the cost of verifying those proofs rises at a slower logarithmic rate than the computation increase. In other words, the computation itself grows slower than the verification cost. As a result, the cost of each transaction within the aggregated set is significantly reduced, making the overall process more cost-effective and accessible for users.
+
+[NOTE]
+====
+In the SHARP and Cairo context, "jobs" refer to the individual Cairo programs or tasks submitted by different users. These jobs contain specific logic or computations that must be executed on the Starknet network.
+====
+
+Additionally, the SHARP allows smaller users with limited computation to benefit from joining other jobs and share the cost of generating the proofs. This collaborative approach is similar to using public transportation instead of a private car, where the cost is distributed among all participants, making it more affordable for everyone.
+
+== Recursive Proofs in the SHARP
+
+One of the most exciting aspects of the SHARP is its use of recursive proofs. Instead of sending the generated proofs directly to the Solidity Verifier, the proofs are first sent to a STARK Verifier program written in Cairo. This STARK Verifier then generates a new proof that the initial proofs were verified. These can be sent back into the SHARP and the STARK Verifier.
+
+This process continues recursively, with each new proof being sent to the Cairo Verifier until a trigger is reached. At this point, the last proof in the series is sent to the Solidity Verifier on Ethereum. This approach allows for greater parallelization of the computation and reduces the time and cost associated with generating and verifying proofs.
+
+At first glance, recursive proofs may seem more complex and time-consuming. However, there are several benefits to this approach:
+
+. *Parallelization*: Recursive proofs allow for work parallelization, reducing user latency and improving SHARP efficiency.
+. *Lower cloud costs*: Since each job is shorter, the required memory for processing is reduced, resulting in lower cloud costs.
+. *Optimization*: Recursive proofs enable the SHARP to optimize for various factors, including latency, on-chain costs, and time to proof.
+
+[NOTE]
+====
+Latency in Starknet encompasses the time taken for processing, confirming, and including transactions in a block. It is affected by factors like network congestion, transaction fees, and system efficiency. Minimizing latency ensures faster transaction processing and user feedback.
+
+Time to proof, however, specifically pertains to the duration required to generate and verify cryptographic proofs for transactions or operations.
+====
+
+== SHARP Backend Architecture and Data Pipeline
```

Comment:
You mentioned the important parts, there are also 3 more services involved. I'm not sure you need to mention them, just making sure you omit them intentionally-  
Job creator 
Blockchain writer (see my comment about dispatcher)
Catcher - monitor blockchain transactions to see that transactions were accepted